mesonbuild/cargo/interpreter.py

```diff
@@ -366,7 +365,7 @@ def _create_project(cargo: Manifest, build: builder.Builder, env: Environment) -
         # Always assume that the generated meson is using the latest features
         # This will warn when when we generate deprecated code, which is helpful
         # for the upkeep of the module
-        'meson_version': build.string(f'>= {env.coredata.version}'),
+        'meson_version': build.string(f'>= {coredata.stable_version}'),
```

Comment:
This seems backwards to me, if the Cargo module makes use of a feature introduced in the dev cycle (which seems possible), then it will warn about that?

Reply:
In a dev cycle, stable_version is next stable version. So it's currently `1.4.0`.